themes/default/content/blog/introducing-the-puluminaries/index.md

```diff
@@ -0,0 +1,103 @@
+<img src="puluminaries.png" width="500" height="500" alt="Puluminaries logo"/>
+
+The Puluminaries program was created to recognize members of the Pulumi community who are experts and leaders in the field of programming and infrastructure, but also who give back to the community to make it grow! They help others get started, give feedback to make Pulumi's roadmap even better, and educate people through content or public speaking. The program is open to all members of the community, and we welcome anyone to join the program.
+
+While the Puluminaries themselves have been active in the community for quite some time, we officially launched the program at [PulumiUP](https://pulumi.com/pulumi-up) in 2022, and it is our pleasure to introduce the first cohort of members to the program!
+
+## Meet The Puluminaries!
+
+### Engin Diri
+
+<img src="engin-diri.png" width="200" height="200" alt="Engin Diri"/>
+<a data-track="twitter" href="https://twitter.com/_ediri"><i class="fab fa-twitter"></i></a>&nbsp;<a data-track="github" href="https://github.com/dirien"><i class="fab fa-github"></i></a>
+
+Engin's excitement for Pulumi is not possible to be measured using current science. He will always share his thoughts on new developements in the Pulumi project, as well as join in for fun and discussion in various PulumiVision livestreams!
+
+### Joshua Studt
+
+<img src="joshua-studt.png" width="200" height="200" alt="Joshua Studt"/>
+<a data-track="github" href="https://github.com/orionstudt"><i class="fab fa-github"></i></a>
+
+Joshua is a longtime member of the Pulumi community, and he contributed to the implementation of Automation API for C#.
+
+### Kat Morgan
+
+<img src="kat-morgan.png" width="200" height="200" alt="Kat Morgan"/>
+<a data-track="twitter" href="https://twitter.com/usrbinkat"><i class="fab fa-twitter"></i></a>&nbsp;<a data-track="github" href="https://github.com/usrbinkat"><i class="fab fa-github"></i></a>&nbsp;<a data-track="linkedin" href="https://linkedin.com/in/usrbinkat"><i class="fab fa-linkedin"></i></a>
+
+Kat has been a helpful member of the Pulumi community - not only providing feedback on real-world use of Pulumi, but also sharing her knowledge with [workshops and webinars](https://konghq.com/webinars/devmyops-deploy-kong-with-pulumi?utm_souce=pulumi&utm_medium=pulumi)
+
+### Komal Ali
+
+<img src="komal-ali.png" width="200" height="200" alt="Komal Ali"/>
+<a data-track="twitter" href="https://twitter.com/zwitkali"><i class="fab fa-twitter"></i></a>&nbsp;<a data-track="github" href="https://github.com/komalali"><i class="fab fa-github"></i></a>&nbsp;<a data-track="linkedin" href="https://linkedin.com/in/komal-ali"><i class="fab fa-linkedin"></i></a>
+
+In addition to being a Puluminary, Komal is also a Pulumni, having spent time working to make Pulumi a better tool for developers -  they were ever-present in making our Python SDK better and championing [Automation API](https://www.pulumi.com/docs/guides/automation-api/)!
```

Comment:
Didn't Komal just straight up build Auto API? Championing seems like a weak term for what was a much greater accomplishment.

Reply:
yeah, I had said something to that effect, but "championing" was what paul stack suggested from his experience?

Reply:
I was using the terminology @stack72 gave me, but if it's accurate to say that they built it themselves that way, we can update. I think there is a way to get both in there though.